### PR TITLE
Create everydaycodings.json

### PR DESCRIPTION
**Pull request by everydaycodings**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2881"></a>

### CODE REVIEW

File: data/everydaycodings.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      "icon": "FaKaggle"
+    },
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/everydaycodings",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://twitter.com/everydaycodings",
```
Comment (Aadarsh805) [inline data/everydaycodings.json:40]: this should be your Instagram link instead of Twitter link

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/everydaycodings.json (hunk 1 above)
Comment (kumarsonsoff3) [inline data/everydaycodings.json:40]: Small typo there! Please update it with your Instagram URL. 🛠

**Review by kumarsonsoff3 — APPROVED** [review]

File: data/everydaycodings.json (hunk 1 above)
Comment (everydaycodings) [reply data/everydaycodings.json:40]: Oh i am sorry my bad.

**Review by Aadarsh805 — APPROVED** [review]
All good 👍

File: data/everydaycodings.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "Kumar Saksham(Everydaycodings)",
+  "displayStatsPublic": false,
+  "type": "personal",
+  "bio": "Freelance Python Developer | **Aspiring Machine Learning Engineer**| Medium Writer",
+  "avatar": "https://github.com/everydaycodings.png",
+  "tags": ["Python Developer", "Machine Learning Engineer", "Tech Writer", "Open Source"],
+  "socials": [
+    { "icon": "FaLinkdin", "url": "https://www.linkedin.com/in/kumar-saksham1891" },
```
Comment (eddiejaoude) [inline data/everydaycodings.json:9]: I think this icon is not valid?

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍 

I think one of the icons needs updating, but this will fallback to a default icon if not found. Updates can come in another pull request

**Pull request merged**